src/components/Stepper/index.scss

```diff
@@ -174,3 +180,7 @@ input.#{$baseClass} {
   margin: auto;
   opacity: 0;
 }
+.load {
+  position: absolute;
+  bottom: 150px;
```

Comment:
are you sure this bottom value is correct?

Reply:
yes, I am sure.